main.go

```diff
@@ -88,8 +90,30 @@ func main() {
 		}
 		c = kubernetes.NewForConfigOrDie(config)
 
+		// Validate arguments are present
+		if len(namespace) == 0 {
+			return fmt.Errorf("Required argument not present: --namespace")
+		}
+		if len(key) == 0 {
+			return fmt.Errorf("Required argument not present: --key")
+		}
+		if len(path) == 0 {
+			return fmt.Errorf("Required argument not present: --path")
+		}
+		if len(name) == 0 {
+			return fmt.Errorf("Required argument not present: --name")
```

Comment:
By convention, returned errors should all start with lower-cased letters, unless the word is a proper name:
```suggestion
			return fmt.Errorf("required argument not present: --namespace")
		}
		if len(key) == 0 {
			return fmt.Errorf("required argument not present: --key")
		}
		if len(path) == 0 {
			return fmt.Errorf("required argument not present: --path")
		}
		if len(name) == 0 {
			return fmt.Errorf("required argument not present: --name")
```